pkgs/by-name/kr/krename/package.nix

```
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  fetchurl,
+  fetchFromGitLab,
+  lib,
+  extra-cmake-modules,
+  kdePackages,
+  taglib,
+  exiv2,
+  podofo,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krename";
+  version = "5.0.60";
```

i think we should add an unstable tag to the version number
```suggestion
  version = "5.0.60-unstable-5ad5f5a";
```